src/symbol/mod.rs

```diff
@@ -575,6 +583,41 @@ impl<'t> TryFromCtx<'t, SymbolKind> for AnnotationReferenceSymbol<'t> {
     }
 }
 
+/// Reference to a managed procedure symbol (`S_LMANPROC` or `S_GMANPROC`).
+///
+/// Symbol kind `S_TOKENREF`.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub struct TokenReferenceSymbol<'t> {
+    /// SUC of the name.
+    pub sum_name: u32,
+    /// Symbol index of the referenced [`ManagedProcedureSymbol`].
+    ///
+    /// Note that this symbol might be located in a different module.
+    pub symbol_index: SymbolIndex,
+    /// Index of the module in [`DebugInformation::modules`](crate::DebugInformation::modules)
+    /// containing the actual symbol.
+    pub module: Option<usize>,
+    /// Name of the procedure reference.
+    pub name: RawString<'t>,
+}
+
+impl<'t> TryFromCtx<'t, SymbolKind> for TokenReferenceSymbol <'t> {
+    type Error = Error;
+
+    fn try_from_ctx(this: &'t [u8], kind: SymbolKind) -> Result<(Self, usize)> {
+        let mut buf = ParseBuffer::from(this);
+
+        let symbol = TokenReferenceSymbol {
+            sum_name: buf.parse()?,
+            symbol_index: buf.parse()?,
+            module: buf.parse::<u16>()?.checked_sub(1).map(usize::from),
```

Comment:
These could be found in PDBs targeting .NET Framework 4.8 (reference https://github.com/getsentry/pdb/issues/145 and the related PR)

Reply:
I've been playing around with llvm-pdbutil, and my conclusion is that the module index in the record is a 1-based index.
S_PROCREF follows the same pattern in the code, and llvm-pdbutil tells me I have
```
0 | S_PROCREF [size = 20] `main`
    module = 1, sum name = 0, offset = 136
```

Subtracting 1 to get normal zero-based indexing makes sense to me.
To be fair though, this is an argument by analogy, since I don't have any pdb files that contain `S_TOKENREF`

Reply:
I suspected exactly that. It'd be helpful to add a comment.

Reply:
sure, I'll add that to each of the locations